siliconcompiler/core.py

```diff
@@ -3639,11 +3651,11 @@ def read(fd):
 
         ##################
         # 24. Save a successful manifest
-        self.set('flowstatus', step, str(index), 'error', 0)
+        self.set('flowstatus', step, index, 'error', 0)
         self.set('arg', 'step', None, clobber=True)
         self.set('arg', 'index', None, clobber=True)
 
-        self.write_manifest("outputs/" + self.get('design') +'.pkg.json')
+        self.write_manifest(f"outputs/{design}.pkg.json")
```

Comment:
```suggestion
        self.write_manifest(os.path.join("outputs", f"{design}.pkg.json"))
```

While we're fixing path styling, it might be a good idea to use the platform-independent `os.path.join`.